Tests/SwiftCliCoreTests/InputValidatorTest.swift

```diff
@@ -4,25 +4,33 @@ import Nimble
 @testable import SwiftCliCore
 
 class ValidatorTests: XCTestCase {
+
+    private var inputValidator: InputValidator!
+
+    override func setUp() {
+        super.setUp()
+        inputValidator = InputValidator(patternNormalizer: PatternNormalizer(), nestedArrayBuilder: NestedArrayBuilder())
+    }
+
+    override func tearDown() {
+        inputValidator = nil
+        super.tearDown()
+    }
```

Comment:
You shouldn't need to set `inputValidator` to `nil` in the `tearDown`, since it's only within the scope of this test class.